module4/src/com/epam/aggregation/entity/Sentence.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.aggregation.entity;
+
+import java.util.Objects;
+
+public class Sentence {
+    public String sentence;
```

Comment:
You should create sentence using Words. It means that e.g. you should have here the list of words.